test/test_other.py

```diff
@@ -8062,7 +8062,12 @@ def test_wasm_nope(self):
       out = self.run_js('a.out.js', assert_returncode=NON_ZERO)
       self.assertContained('no native wasm support detected', out)
 
-  @requires_wasm_eh
+  # FIXME Node v18.13 (LTS as of Jan 2023) has not yet implemented the new
+  # optional 'traceStack' option in WebAssembly.Exception constructor
+  # (https://developer.mozilla.org/en-US/docs/WebAssembly/JavaScript_interface/Exception/Exception)
+  # and embeds stack traces unconditionally. Change this back to
```

Comment:
I'm note sure its useful but we now have ways to checking the node version used in testing (looks for `shared.check_node_version` in `test/common.py`) and ways of running them (`test-node-compat` in `.circleci/config.yml``).     Maybe not worth the effort to make that work for this case?

Reply:
Our CI is fine without this, because we use node v14, which doesn't have Wasm EH support at all, and [your logic](https://github.com/emscripten-core/emscripten/blob/33a9cf1bceaaf52229285b6e77bf7f5747b18dba/test/common.py#L494-L507) will make sure we fall back to v8. But if we upgrade our node to a version such as v18, this will start to fail on our CI, and who will do the update can be confused by the failures, so I made this changes.

Another thing we can do is we leave `@requires_wasm_eh` as is and leave this comment for future reference, because our CI falls back to v8 and has no problem currently, and deal with the error when we update the node, but the person who's doing the update will still be confused.

Reply:
We now also run some tests of the very latest node tip of tree (see `test-node-compat`).

Reply:
Just FYI in case you want to run some wasm EH under node too... we can now.

Reply:
I tested with several node versions:
- Node 14.18.2 (Our default): No support of Wasm EH
- Node 17.5: Hasn't implemented `traceStack`, so always embeds stack traces. Prints stack traces when thrown and uncaught.
- Node 18.13 (LTS): Same
- Node 19.5 (Current): Has implemented `traceStack`. But now it doesn't show stack traces when exceptions are thrown and uncaught... Not sure why.

Given this a little confusing situation, I'm tempted to use only V8 for this test for now...